README.md

```diff
@@ -130,41 +138,24 @@ Chef Vault provides an easier way to manage Data bags and configure them. To con
 ```
 knife vault create wazuh_secrets api '{"id": "api", "htpasswd_user": "user", "htpasswd_passcode": "password"}' -A "username" -C "manager-1"
 ```
+
 Where `-A` defines the workstation users authorized to modify/edit the vault and `-C` defines the nodes that have access to the defined vault.
 
 After that, the vault will be created and synced with the server. The defined nodes will store the required keys to decrypt the vault content and consume it.
 
 You can check Chef Official Documentation about [Chef Vault](https://docs.chef.io/chef_vault.html) for detailed info.
 
-## Use through Berkshelf
-
-The easiest way to making use of these cookbooks (especially `wazuh_filebeat` & `wazuh_elastic` until they are published to Supermarket) is by including in your `Berksfile` something like the below:
-
-```ruby
-cookbook 'wazuh', gitHub: 'wazuh/wazuh-chef', rel: 'wazuh'
-cookbook 'wazuh_filebeat', gitHub: 'wazuh/wazuh-chef', rel: 'wazuh_filebeat'
-cookbook 'wazuh_elastic', gitHub: 'wazuh/wazuh-chef', rel: 'wazuh_elastic'
-```
-
-This will source all three cookbooks housed in this repo from GitHub.
-
 ## Choose to register an agent into a manager or not
 Now we give the possibility to choose to register an agent after being configured and installed in a manager. 
 
-In order to do so, it's only needed to assign the value `yes` to the variable ` default['ossec']['agent_auth']['register'] ` in the attributes file_ ` wazuh-chef/cookbooks/wazuh_agent/attributes/authd.rb `:
-
-```
-default['ossec']['agent_auth']['register'] = 'yes'
-```
-
-To connect an agent with the manager simply modify the `wazuh-chef/roles/wazuh_agent.json` with the desired IP address:
+To connect an agent with the manager simply modify the `wazuh-chef/roles/wazuh_agent.json` with the 
+manager IP address:
 
 ```
-"registration_address": "<YOUR REGISTRATION IP ADDRESS>",
 "address": "<YOUR MANAGER IP ADDRESS>"
 ```
 
-In other case, we just assign a different value which is not `yes`.
+Since Wazuh 4.0, by default, the agent registers automatically with the manager through enrollment. Configuration details can be found on [Enrollment section](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/client.html#reference-ossec-client).
```

Comment:
`the agent registers automatically with the manager through enrollment` --> `the automatically registers against the manager`